cpp/src/DataStorm/NodeSessionManager.cpp

```diff
@@ -65,7 +71,7 @@ NodeSessionManager::init()
     auto instance = getInstance();
 
     auto forwarder = [self = shared_from_this()](Ice::ByteSeq e, const Ice::Current& c) { self->forward(e, c); };
-    _forwarder = Ice::uncheckedCast<LookupPrx>(instance->getCollocatedForwarder()->add(move(forwarder)));
+    _forwarder = Ice::uncheckedCast<LookupPrx>(instance->getCollocatedForwarder()->add(std::move(forwarder)));
```

Comment:
Preferred style is to use the LookupPrx constructor. I am guessing add returns a non-null proxy?

Reply:
Yes it does, we can probably updates this once we sort out the APIs to use nullable proxies only when required.